tests/client/coll.rs

```diff
@@ -14,7 +12,7 @@ fn find_and_insert() {
 
     // Insert document
     let doc = doc! {
-        "title" => Bson::String("Jaws".to_owned())
+        "title" => (Bson::String("Jaws".to_owned()))
```

Comment:
I can change it to work with any delimeter, but it just has to be different than the braces and brackets used for the arrays and nested documents.

Alternately, I could change it to use a different symbol after (or instead of) "=>", but it would have to be before the actual value to get rid of the parsing ambiguity. (i.e. something like `"key" => $value` could work, with the dollar sign or some other token resolving the ambiguity).